python/vyos/ifconfig/interface.py

```diff
@@ -282,7 +309,7 @@ def get_config(cls):
         """
         return deepcopy(cls.default)
 
-    def __init__(self, ifname, **kargs):
+    def __init__(self, ifname, netns='', **kargs):
```

Comment:
Why is this needed? `netns` key is part of `**kargs` and will thus be available as `self.config['netns']`

Reply:
It also used for  parent class “control.py” as self.netns
Should we somehow get **kargs in the parent class?

Reply:
self.config is accessible in the base class, too

Reply:
Done! Netns removed from def __init__ arg